application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -376,33 +377,37 @@ record TemporaryData(Instant expiresAt, String duration) {
     /**
      * Gives out advice depending on the {@link ModerationAction} and the parameters passed into it.
      *
-     * @param action gives out a {@link ModerationAction} to be able to respond and build a message
-     *        appropriately such as {@link ModerationAction#MUTE} etc.
-     * @param message is the message that is being used to build responses, such as
-     *        {@link Guild#getName()}, and reasoning for an action.
+     * @param action the action that is being performed, such as banning a user.
+     * @param temporaryData is the message that is being used to display action duration.
+     * @param guild to retrieve the guild.
```

Comment:
"give me a guild. so i can retrieve the guild" - makes no sense. ur describing the wrong thing. put urself in the shoes of the user of the method, they want to know what ur doing with this, what is the purpose? which guild should i give u?

`@param guild for which the action was triggered`